### PR TITLE
Learning inmanta js

### PR DESCRIPTION
**Pull request by FloLey**
# Description

* Short description here *

closes *Add ticket reference here*

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

**Comment by FloLey** [comment]
@andraskvr I've added some test, it would clearly not be enough would this not be an exercise but I think it is enough for now. 
Do you agree or would you want me to add some more? Maybe you could do a little review of my code to?

File: src/Slices/ResourceV2/UI/resourceDetailsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  resourceId: string;
+}
+
+export const ResourceDetailsTab: React.FunctionComponent<ResourceTabsProps> = ({
+  resourceId,
+}) => {
+  const { queryResolver } = useContext(DependencyContext);
+  const [activeTab, setActiveTab] = useUrlStateWithString<TabKey>({
+    default: TabKey.Attributes,
+    key: `tab`,
```
Comment (andraskvr) [inline src/Slices/ResourceV2/UI/resourceDetailsTab.tsx:24]: It's ok for now, but the key is not unique on the page, so if you change the active tab at one place you will change it in every row.

File: src/Slices/ResourceV2/UI/resourceDetailsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  Description,
+  PageContainer,
+  ResourceStatusLabel,
+} from "@/UI/Components";
+
+interface ResourceTabsProps {
+  resourceId: string;
+}
+
+export const ResourceDetailsTab: React.FunctionComponent<ResourceTabsProps> = ({
```
Comment (andraskvr) [inline src/Slices/ResourceV2/UI/resourceDetailsTab.tsx:18]: There is also the `React.FC` shorthand instead of `React.FunctionComponent`, both are fine

**Review by andraskvr — COMMENTED** [review]
Some suggestions for the future, but in general it's ok :+1:

File: src/Slices/ResourceV2/UI/resourceDetailsTab.tsx (hunk 1 above)
Comment (FloLey) [reply src/Slices/ResourceV2/UI/resourceDetailsTab.tsx:24]: I see, this I need to remember to change in the future :p

**Pull request closed without merging**